meraki_v0/aio/__init__.py

```diff
@@ -2,84 +2,84 @@
-from .api.link_aggregations import AsyncLinkAggregations
-from .api.mg_dhcp_settings import AsyncMGDHCPSettings
-from .api.mg_lan_settings import AsyncMGLANSettings
-from .api.mg_connectivity_monitoring_destinations import AsyncMGConnectivityMonitoringDestinations
-from .api.mg_port_forwarding_rules import AsyncMGPortForwardingRules
-from .api.mg_subnet_pool_settings import AsyncMGSubnetPoolSettings
-from .api.mg_uplink_settings import AsyncMGUplinkSettings
-from .api.mr_l3_firewall import AsyncMRL3Firewall
-from .api.mv_sense import AsyncMVSense
-from .api.mx_1_1_nat_rules import AsyncMX11NATRules
-from .api.mx_1_many_nat_rules import AsyncMX1ManyNATRules
-from .api.mx_l3_firewall import AsyncMXL3Firewall
-from .api.mx_l7_application_categories import AsyncMXL7ApplicationCategories
-from .api.mx_l7_firewall import AsyncMXL7Firewall
-from .api.mx_vlan_ports import AsyncMXVLANPorts
-from .api.mx_vpn_firewall import AsyncMXVPNFirewall
-from .api.mx_cellular_firewall import AsyncMXCellularFirewall
-from .api.mx_inbound_firewall import AsyncMXInboundFirewall
-from .api.mx_port_forwarding_rules import AsyncMXPortForwardingRules
-from .api.mx_static_routes import AsyncMXStaticRoutes
-from .api.mx_warm_spare_settings import AsyncMXWarmSpareSettings
-from .api.malware_settings import AsyncMalwareSettings
-from .api.management_interface_settings import AsyncManagementInterfaceSettings
-from .api.meraki_auth_users import AsyncMerakiAuthUsers
-from .api.monitored_media_servers import AsyncMonitoredMediaServers
-from .api.named_tag_scope import AsyncNamedTagScope
-from .api.netflow_settings import AsyncNetFlowSettings
-from .api.networks import AsyncNetworks
-from .api.openapi_spec import AsyncOpenAPISpec
-from .api.organizations import AsyncOrganizations
-from .api.pii import AsyncPII
-from .api.radio_settings import AsyncRadioSettings
-from .api.saml_roles import AsyncSAMLRoles
-from .api.sm import AsyncSM
-from .api.snmp_settings import AsyncSNMPSettings
-from .api.ssids import AsyncSSIDs
-from .api.security_events import AsyncSecurityEvents
-from .api.splash_login_attempts import AsyncSplashLoginAttempts
-from .api.splash_settings import AsyncSplashSettings
-from .api.switch_acls import AsyncSwitchACLs
-from .api.switch_port_schedules import AsyncSwitchPortSchedules
-from .api.switch_ports import AsyncSwitchPorts
-from .api.switch_profiles import AsyncSwitchProfiles
-from .api.switch_settings import AsyncSwitchSettings
-from .api.switch_stacks import AsyncSwitchStacks
-from .api.syslog_servers import AsyncSyslogServers
-from .api.traffic_analysis_settings import AsyncTrafficAnalysisSettings
-from .api.traffic_shaping import AsyncTrafficShaping
-from .api.uplink_settings import AsyncUplinkSettings
-from .api.vlans import AsyncVLANs
-from .api.webhook_logs import AsyncWebhookLogs
-from .api.wireless_health import AsyncWirelessHealth
-from .api.wireless_settings import AsyncWirelessSettings
-from ..config import (
+from meraki_v0.aio.rest_session import *
```

Comment:
Why does this one line import from meraki_v0.aio and the rest here only import from meraki_v0? (to be clear -- I think this line is correct, and it's the others worth examining)

Reply:
Location meraki_v0.aio.__init__.py
Original 

from .rest_session import *                                                  - This does reference the rest_session file in the same folder

But, for the rest, you're correct, it should be referencing meraki_v0.aio.api not meraki_v0.apo
Will correct